src/code/z_parameter.c

```diff
@@ -2383,29 +2411,34 @@ void Interface_UpdateMagicBar(PlayState* play) {
     s16 borderChangeR;
     s16 borderChangeG;
     s16 borderChangeB;
-    s16 temp;
-
-    switch (gSaveContext.unk_13F0) {
-        case 8:
-            temp = gSaveContext.magicLevel * 0x30;
-            if (gSaveContext.unk_13F4 != temp) {
-                if (gSaveContext.unk_13F4 < temp) {
-                    gSaveContext.unk_13F4 += 8;
-                    if (gSaveContext.unk_13F4 > temp) {
-                        gSaveContext.unk_13F4 = temp;
+    s16 temp; // target for magicCapacity or magicBorderIndex
```

Comment:
```suggestion
    s16 temp; // target for magicCapacity, or magicBorderIndex
```
as in `(target for magicCapacity) or (magicBorderIndex)`?
instead of `target for (magicCapacity or magicBorderIndex)` 😎